app/controllers/editions_controller.rb

```diff
@@ -112,9 +126,8 @@ def create_with_new_work
     render 'new'
   end
 
-  def create_with_existing_work
-    @work = Work.friendly.find(params.require(:existing_work).permit(:id)[:id])
-    @edition.work_id = @work.id
+  def create_with_existing_work(work:)
```

Comment:
No other function currently uses keyword arguments, use a regular param instead (same for the other new functions)

Reply:
Done